rust/lancedb/Cargo.toml

```diff
@@ -40,6 +40,8 @@ serde = { version = "^1" }
 serde_json = { version = "1" }
 # For remote feature
 reqwest = { version = "0.11.24", features = ["gzip", "json"], optional = true }
+polars-arrow = { version = "0.39.2", optional = true }
+polars = { version = "0.39.2", optional = true}
```

Comment:
We don't want to force consumers to be locked to our version of polars. This should be wildcarded, or at least specify a wider range similar to what polars does with `arrow` dependencies

Reply:
Good call. Updated to >= 0.38. There were some updates to how polars handled strings/string views in 0.38.

Reply:
Actually, since Polars isn’t at version 1 yet, technically anything can change between version numbers. 0.40 or 0.39.3 aren’t guaranteed to be backwards compatible. So, maybe expecting a specific version (0.39.2) is the way to go. Not sure. What do you think?

Reply:
From reading through the code, it looks like we are only using functionality that is "stablilzed". Those api's haven't changed much _(if at all)_ since very early on. So overall, I'd say it's pretty safe to use a range like `>= 0.38` here.